Makefile

```diff
@@ -1,7 +1,19 @@
 MAKEFLAGS += --no-print-directory
 COV_FLAGS :=
 SHELL:=/bin/bash
+NPROC:=$(shell python -c "import multiprocessing as mp; print(mp.cpu_count())")
+
+# Build variables
+SR_BUILD := Release
+SR_LINK := Shared
+SR_TEST_REDIS_MODE := Clustered
+SR_TEST_RAI_VER := 1.2.7
+SR_WLM := Local
+SR_WLM_FLAGS :=
```

Comment:
It will be used in the next stage, where we automate creation/teardown of Redis with the `test` target. I could remove it for now, but it will jsut be coming right back...

Reply:
renamed to SR_TEST_...